src/main/java/org/springframework/cloud/deployer/spi/cloudfoundry/CloudFoundryDeployerProperties.java

```diff
@@ -24,10 +24,13 @@
 import org.springframework.boot.context.properties.ConfigurationProperties;
 
 /**
+ * Holds configuration properties for connecting to a Cloud Foundry runtime.
+ *
+ * @author Eric Bottard
  * @author Greg Turnquist
  */
 @ConfigurationProperties(prefix = "deployer.cf")
-public class CloudFoundryAppDeployProperties {
+public class CloudFoundryDeployerProperties {
```

Comment:
Not related to this rename, but using Set in the property for the default services to bind is not very YML friendly, see http://yaml.org/type/set.html   We should make this a list.